source/docs/essentials/extend-the-theme.md

```diff
@@ -48,8 +49,8 @@ Then, we need to tell Front-Commerce that this module exists in your [`.front-co
 module.exports = {
   name: "Front Commerce",
   url: "http://www.front-commerce.test",
--  modules: [],
-+  modules: ["./my-module"],
+-  modules: [, "./src"],
```

Comment:
Please remove the `,` here.

If we're adding the `./src` I think we should add a note that it will be there in case people are using the skeleton only.